saleor/graphql/order/tests/queries/test_pagination.py

```diff
@@ -150,7 +150,7 @@ def draft_orders_for_pagination(db, channel_USD):
             3,
         ),
         ({"created": {"gte": str(date.today() - timedelta(days=3))}}, [3.0, 2.0], 3),
-        ({"created": {"lte": str(date.today())}}, [0.0, 3.0], 4),
+        ({"created": {"lte": str(date.today())}}, [3.0, 2.0], 4),
```

Comment:
Since I changed ordering from `number` field to `updated_at`, test had to be aligned.